sceptre/sageit/templates/gh_deployer.yaml

```diff
@@ -0,0 +1,62 @@
+Description: Setup IAM service user and policies for Github deployment of S3 website
```

Comment:
A better place to create service accounts is at https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/600-access/_tasks.yaml#L4